src/beast/beast/http/parser.h

```diff
@@ -100,8 +100,6 @@ class parser
         m_.method = method;
         m_.url = url;
         m_.version = major * 10 + minor;
-        m_.keep_alive = keep_alive;
-        m_.upgrade = upgrade;
```

Comment:
I've been trying to figure out why these are no longer needed or who sets them instead. The previous version looks correct to me, but I'm clearly confused.